tests/dragonfly/cluster_test.py

```diff
@@ -2019,6 +2018,71 @@ async def test_cluster_migration_huge_container(df_factory: DflyInstanceFactory)
     assert source_data == target_data
 
 
+@dfly_args({"proactor_threads": 2, "cluster_mode": "yes"})
+@pytest.mark.asyncio
+async def test_cluster_migration_huge_container_while_seeding(
+    df_factory: DflyInstanceFactory, df_seeder_factory: DflySeederFactory
+):
+    instances = [
+        df_factory.create(
+            port=next(next_port),
+            admin_port=next(next_port),
+            serialization_max_chunk_size=1_000,
+        )
+        for _ in range(2)
+    ]
+    df_factory.start_all(instances)
+
+    nodes = [await create_node_info(instance) for instance in instances]
+    nodes[0].slots = [(0, 16383)]
+    nodes[1].slots = []
+    client0 = nodes[0].client
+    client1 = nodes[1].client
+
+    await push_config(json.dumps(generate_config(nodes)), [node.admin_client for node in nodes])
+
+    async def get_memory(client, field):
+        info = await client.info("memory")
+        return info[field]
+
+    logging.debug("Seeding cluster")
+    seeder = df_seeder_factory.create(
+        keys=100, port=instances[0].port, cluster_mode=True, mirror_to_fake_redis=True
+    )
+    seed = asyncio.create_task(seeder.run())
```

Comment:
first call 
await seeder.run(target_deviation=0.1)
To fill the data
and than 
seed = asyncio.create_task(seeder.run())

you will not need the sleep in line 2054 after this change